packages/@uppy/dashboard/package.json

```diff
@@ -56,6 +56,7 @@
   },
   "devDependencies": {
     "@uppy/core": "workspace:^",
+    "@uppy/dropbox": "workspace:^",
```

Comment:
you're putting dropbox in here but you actually use remote-sources in the test, causing the build to fail. Use one or the other